torch/nn/modules/rnn.py

```diff
@@ -258,6 +274,12 @@ def extra_repr(self) -> str:
             s += ', bidirectional={bidirectional}'
         return s.format(**self.__dict__)
 
+    def __getstate__(self):
+        # Don't serialize the weight references.
+        state = self.__dict__.copy()
```

Comment:
should this be `state = super().__getstate__()` ?

Reply:
I tried that first, but apparently `super().__getstate__()` didn't exist until Python 3.11 for some reason.

AFAICT from the python docs, the default behavior when a `__getstate__()` impl is not present is equivalent to returning the object's `state_dict`.